docs/04-migrate-to-parts.md

```
@@ -9,17 +9,17 @@ You can read the API reference for the `parts` property [here](https://sdk.verce
 Your existing project must already have messages stored in the database. To migrate your messages to use `parts`, you will have to create new tables for `Message` and `Vote`, backfill the new tables with transformed messages, and delete (optional) the old tables.
 
 These are the following steps:
-1. Create tables `Message_v2` and `Vote_v2` with updated schemas at `/lib/db/schema.ts`
+1. Create tables `Message_v2` and `Vote_v2` with updated schemas at `/server/db/schema.ts`
```

markdownlint-cli2 (0.17.2)

Lists should be surrounded by blank lines
null

(MD032, blanks-around-lists)